vspace/src/main/java/edu/asu/diging/vspace/core/services/IStaffSearchManager.java

```diff
@@ -0,0 +1,26 @@
+package edu.asu.diging.vspace.core.services;
+
+import java.util.HashSet;
+
+import edu.asu.diging.vspace.core.model.IVSpaceElement;
+
+public interface IStaffSearchManager {
+
+    HashSet<IVSpaceElement> searchInSpaces(String search,int page);
```

Comment:
parameter name should be more descriptive like `query` or `searchTerm` or something like that